server/config.example.yaml

```diff
@@ -61,11 +61,11 @@ email:
   pool_size: 4
 
 openim:
-  secret: openkf
+  secret: openIM123
   ip: 127.0.0.1
   api_port: 10002
   platform_id: 5 # web
-  admin_id: openkf_admin # OpenIMServer admin userID
+  admin_id: openIMAdmin
```

Comment:
this is integration with the im part?

Reply:
It's easy to use in default openim server config.